extensions/amp-list/amp-list.md

```diff
@@ -153,11 +153,11 @@ In `<amp-list>`, you can use the [`items`](#items) attribute to render a subset
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. Add an element with the AMP `overflow` attribute to `amp-list` if the AMP framework is unable to size it as requested. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if one of the following conditions are met:
+Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if all of the following conditions are met:
```

Comment:
maybe replace the deleted line and the last line with: "AMP will display this element if all of the following conditions are met:"

(since we don't have to repeat the idea that you have to include this child element)

Reply:
Done!